hazelcast-tpc-engine/src/main/java/com/hazelcast/internal/tpcengine/Reactor.java

```diff
@@ -322,6 +323,64 @@ public final boolean awaitTermination(long timeout, TimeUnit unit) throws Interr
      */
     public abstract void wakeup();
 
+    /**
+     * Executes a Callable on the Reactor and returns a CompletableFuture with
+     * its content.
+     * <p/>
```

Comment:
```suggestion
     * <p>
```

Nit - I _think_ <p> is the standard, e.g. [Oracle docs](http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#format).